bridges/opentracing/lib/opentelemetry/bridge/opentracing.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'opentracing'
+require 'opentelemetry'
+
+module OpenTelemetry
+  module Bridge
+    # OpenTracing Bridge provides a means of converting
+    # from OpenTelemetry to OpenTracing objects
+    OT = OpenTracing
```

Comment:
This is a bit of a weird way to access the "other" `OpenTracing` namespace. You can instead (below):
```ruby
FORMAT_TEXT_MAP = ::OpenTracing::FORMAT_TEXT_MAP
FORMAT_RACK = ::OpenTracing::FORMAT_RACK
```

Reply:
Ah ok thanks